meson.build

```diff
@@ -56,6 +56,8 @@ cdata.set_quoted('GAEGULI_PIPELINE_WORKER',
   join_paths(gaeguli_bindir, 'gaeguli_pipeline_worker'))
 cdata.set_quoted('GAEGULI_TARGET_WORKER',
   join_paths(gaeguli_bindir, 'gaeguli_target_worker'))
+cdata.set('GAEGULI_PIPELINE_WORKER_ARGS_NUM', 8)
+cdata.set('GAEGULI_TARGET_WORKER_ARGS_NUM', 11)
```

Comment:
Why put these two into `meson.build`? Can't they be defined in `gaeguli-internal.h` for instance?